.github/workflows/hive-ci.yml

```diff
@@ -82,6 +82,8 @@ jobs:
       SPARK_LOCAL_IP: localhost
     steps:
     - uses: actions/checkout@v2
+    - name: Update update-ca-certificates
```

Comment:
do we need this?

Reply:
i had issue yesterday on my git actions on some strange error link to SSL. Sorry i forgot to remove it